Review.java

```diff
@@ -105,7 +105,32 @@ public static double totalSentiment(String fileName)
      return reviewscore; 
   }
 
-  
+ public static String fakeReview(String fileName)
+  {
+      String str = textToString(fileName);
+      String currWord = "";
+      String finalStr = "";
+      for(int i =0; i < str.length();i++)
+      {
+        if(!str.substring(i,i+1).equals(" "))

```

Comment:
why do you have an ! right here? otherwise it looks good

Reply:
the ! is to check if its not equal to " ".